tests/devices/oav/test_snapshots.py

```diff
@@ -116,8 +116,8 @@ def image_data_coro(image_bytes: BytesIO) -> AsyncMock:
 
 
 def assert_images_identical(left: Image.Image, right: Image.Image):
-    left_data = left.getdata()
-    right_data = right.getdata()
+    left_data = list(left.getdata())  # type:ignore # Can remove once https://github.com/python-pillow/Pillow/pull/9261 is released
```

Comment:
Have commented on the PR but I think the simpler change that doesn't skip type checking would be 


```suggestion
    left_data = list(iter(left.getdata()))
```

Reply:
Although looking at what the test is doing, I think the whole thing could be 

```python
assert all(l == r for l,r in zip(left.getdata(), right.getdata()), strict=True))
```